packages/schemas/src/seeds/management-api.ts

```diff
@@ -0,0 +1,106 @@
+import { generateStandardId } from '@logto/core-kit';
+
+import type { CreateResource, CreateRole, CreateScope } from '../db-entries/index.js';
+import { PredefinedScope, UserRole } from '../types/index.js';
+import { adminTenantId, defaultTenantId } from './tenant.js';
+
+export type AdminData = {
+  resource: CreateResource;
+  scope: CreateScope;
+  role: CreateRole;
+};
+
+// Consider remove the dependency of IDs
+const defaultResourceId = 'management-api';
+const defaultScopeAllId = 'management-api-all';
+
+// Consider combine this with `createManagementApiInAdminTenant()`
+/** The fixed Management API Resource for `default` tenant. */
+export const defaultManagementApi = Object.freeze({
+  resource: {
+    tenantId: defaultTenantId,
+    /** @deprecated You should not rely on this constant. Change to something else. */
+    id: defaultResourceId,
+    /**
+     * The fixed resource indicator for Management APIs.
+     *
+     * Admin Console requires the access token of this resource to be functional.
+     */
+    indicator: `https://${defaultTenantId}.logto.app/api`,
```

Comment:
Wondering if it's OK to hard code the domain name here, but that is and old issue and doesn't need to be handled in this PR.

Reply:
would you like to elaborate the concern here? i was thinking the alignment between the indicator and the actual URL is a "nice-to-have" thing

Reply:
Well, some of the OSS users may want to align this to their actual domain, can we upgrade that by adding it to envset in the future? Not a blocker.

Reply:
good call, let's consider this until user asks, haha

Reply:
> A possible workaround for this case might just be updating it in DB after seeding?

`koaAuth()` needs update too.